### ISSUE TITLE
Make error pages more user friendly

### ISSUE THREAD

**Issue by sfkeller** [geometalab/osmaxx#789]
There will be download weblinks out in the wild (especially expired ones) which lead to an error page 

    404 Not Found 
    ----
    nginx

Example: https://osmaxx.hsr.ch/media/osmaxx/outputfiles/bee7e88d-f093-4077-a533-1cc983962039/rapperswil-jona_wgs-84_2017-01-19_gpkg_full-detail.zip

Similar to this there's another error page occurring with wrong pages. 

    Not Found
    The requested URL /x was not found on this server.

Example: https://osmaxx.hsr.ch/x

**Comment by das-g** [comment]
Good point. Something we didn't yet think of (or maybe wasn't even relevant yet) in #102 / #107
